Resources/Locale/en-US/_DV/silicons/lawboard-insertion.ftl

```diff
@@ -0,0 +1,3 @@
+lawboard-insertion-needs-panel-open = { CAPITALIZE($target) } needs { POSS-ADJ($target) } maintenance panel opened first.
```

Comment:
there is almost certainly another loc string that is this verbatim

Reply:
it is but it's bad localization practice to reuse a key for different things just because the string happened to be the same at a given time